analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.vb

```diff
@@ -0,0 +1,242 @@
+﻿Imports System
```

Comment:
Not needed. VB has default imported namespaces from year 2002 or something like that automatically
```suggestion
```